bundles/org.openhab.binding.bondhome/src/main/resources/OH-INF/thing/Channels.xml

```diff
@@ -0,0 +1,302 @@
+		<item-type>Switch</item-type>
+		<label>Start Increasing Up Light Brightness</label>
+		<description>Starts or stops increasing the brightness of the ceiling fan up light.</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="upLightDimmerDcrChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Start Decreasing Up Light Brightness</label>
+		<description>Starts or stops decreasing the brightness of the ceiling fan up light.</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="downLightChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Down Light</label>
+		<description>Turns on the down light.</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+			<tag>Switchable</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="downLightEnableChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Enable Down Light</label>
+		<description>Enables or disables the down light of the ceiling fan. The light must also be on to turn on the down
+			light.</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+			<tag>Switchable</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="downLightBrightnessChannelType" advanced="false">
+		<item-type>Dimmer</item-type>
+		<label>Down Light Brightness</label>
+		<description>Adjusts the brightness of the fan down light</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="downLightDimmerStartStopChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Start Dimming Down Light</label>
+		<description>Starts or stops changing the brightness of the ceiling fan down light. The direction of dimming cannot be
+			controlled.</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="downLightDimmerIncrChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Start Increasing Down Light Brightness</label>
+		<description>Starts or stops increasing the brightness of the ceiling fan down light.</description>
+		<category>Light</category>
+	</channel-type>
+
+	<channel-type id="downLightDimmerDcrChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Start Decreasing Down Light Brightness</label>
+		<description>Starts or stops decreasing the brightness of the ceiling fan down light.</description>
+		<category>Light</category>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="stopChannelType" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Stop</label>
+		<description>Stops any ongoing change actions</description>
+		<category>Switch</category>
+		<tags>
+			<tag>Switchable</tag>
+		</tags>
+	</channel-type>
```

Comment:
These channels look like to send a 1 time command event to the device. These could be better modelled as 1 channel with command options. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#command-description
This might apply for other channels too.